third_party/gtest/internal/gtest-port-arch.h

```diff
@@ -35,80 +35,210 @@
 #ifndef GOOGLETEST_INCLUDE_GTEST_INTERNAL_GTEST_PORT_ARCH_H_
 #define GOOGLETEST_INCLUDE_GTEST_INTERNAL_GTEST_PORT_ARCH_H_
 
+// Seems to be an internal undefined variable for us.
```

Comment:
This change creates discrepancy between the upstream version and our copy complicating the process of updating gtest. To me it seems that a better way to handle this would be disabling the check for gtest component as we don't use it to build the library. Do you see any problems with this?

Reply:
Discussed offline. A trick with removing an option worked fine. gtest.h is still modified since it's included in oneDNN gtests. To avoid the change, a single entry point to gtest.h must be done and a proxy header technique, same as one used for TBB, should be applied there, but it will require whole gtest folder update, so I decided to skip on it this time.